lib/element/locator.js

```diff
@@ -159,13 +162,7 @@ class LocateElement {
       value = null;
     }
 
-    let parsedValue = value && this.transport.getElementId(value);
-
-    return {
-      status: 0,
-      // for recursive lookups, the value is already parsed
-      value: Utils.isUndefined(parsedValue) ? value : parsedValue
-    };
+    return value;
```

Comment:
Most of the other changes (e.g. the `toElement` in the transport or various `getElementId`) are now required because of the changed return value here. It now returns the "unprocessed" response (e.g. `{ ELEMENT: '...' }`) and not just the element ID directly.